bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -7,14 +7,27 @@ class BashGastToCodeConverter(AbstractGastToCodeConverter):
     pretty_name = "Bash"
    
     def handle_bool(gast):
-        if gast["value"] == 1:
-            return "true"
-        else:
-            return "false"
+        return "Bash does not support booleans"
```

Comment:
I say we just return the strings "true" and "false" anyways since they end up working the same way.

Reply:
Okay